src/warnet/dashboard.py

```diff
@@ -0,0 +1,11 @@
+import click
+
+
+@click.command()
+def dashboard():
```

Comment:
meh on having a command for this. seems easier to just prompt the user in the logs? also this assumes port forwarding, which I don't think will always be the case, so this command wont work if we move to having proper ingress. lastly, id assume this webbrowser thing will end up failing for some weird reason on someone's computer (cough cough windows).